git_machete/cmd.py

```diff
@@ -3696,7 +2365,7 @@ def check_required_param(in_args: List[str], allowed_values_string: str) -> str:
             cb = current_branch(cli_ctxt)
             dest = machete_client.parse_direction(param, cb, allow_current=False, down_pick_mode=True)
             if dest != cb:
-                go(cli_ctxt, dest)
+                git_operations.go(cli_ctxt, dest)
```

Comment:
While you're here, I'd change the name of this method from `go` (which is a machete-specific term) to `checkout` (which is machete-agnostic)